docker-compose.yml

```
@@ -0,0 +1,16 @@
+version: "3.4"
+services:
+  postgres:
+    image: postgres:latest
+    container_name: fitflow-postgres
+    ports:
+      - "5432:5432"
+    expose:
+      - "5432"
+    volumes:
+      - ./postgres_data:/var/lib/postgresql/data
```

Acho melhor essa pasta `postgres_data` ser uma pasta oculta e seguir o padrão [kemac-kase](https://www.freecodecamp.org/news/programming-naming-conventions-explained/). O que acha?

```suggestion
      - ./.postgres-data:/var/lib/postgresql/data
```

---

sem problemas, acho que é ótimo assim tmb.